src/interfaces/networkHealthConfig.interface.ts

```diff
@@ -0,0 +1,8 @@
+export interface NetworkHealthConfig {
+  numberOfBlocks: number
```

Comment:
nitpick(blocking): be more precise. What kind of blocks? `numberOfExpectedBLocks`?